pkg/cmd/pipeline/testdata/taskrunspec.yaml

```diff
@@ -0,0 +1,6 @@
+- pipelineTaskName: unit-test-task
```

Comment:
I referred [#1088](https://github.com/tektoncd/cli/pull/1088) writing this pr, and it's mostly the same but here hardcoded pipelineTaskName maybe seems fragile to future change. maybe ok?

Reply:
@haedaal what could be the other options ? multiple flags `--tasrun-spec`, one fore each task ? what else ?
I think the one is ok, but I won't if there could be other alternative to look into.

Reply:
may be we can add a check in parsing to have multiple tasks

Reply:
@vdemeester I feel like there's no better alternatives, as it looks exactly same as k8s spec. I think simplicity is worth a bit of fragility.